scripts/postgres_setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Thing shim around the normal Docker postgres entrypoint that allows us to run
+# non-application migrations. Things like DB and user creations that would be
+# done by cloud-automation tasks in a normal env.
+
+set -e
+
+# Initialize the DB, but don't allow outside connections yet.
+docker-entrypoint.sh postgres -c listen_addresses='127.0.0.1' &
```

Comment:
This relates to running the script _after_ the init scripts but _before_ the DB is accessible to other apps on the network. That distinction is significant because our apps (mostly) wait for the DB to be available and once it's online they get to work.